skorch/callbacks/logging.py

```diff
@@ -70,31 +78,42 @@ class PrintLog(Callback):
       The number formatting. See the documentation of the ``tabulate``
       package for more details.
 
+    stralign : str (default='right')
+      The alignment of columns with strings. Can be 'left', 'center',
+      'right', or ``None`` (disable alignment). Default is 'right' (to
+      be consistent with numerical columns).
+
     """
     def __init__(
             self,
             keys_ignored='batches',
```

Comment:
We could do this, especially since users will always want this. However, that change doesn't need to be in this PR, as the problem exists independently.

Reply:
Sure, I'll create an issue that documents this.